### DISCUSSION TITLE
How to resize a window?

### DISCUSSION THREAD

**Discussion by nikitablack** [emilk/egui#465] [Q&A]
Hello. Somehow I can't figure out how to make an empty resizable window. This code: 
```
egui::Window::new("My Window")
                    .show(ctx, |ui| {
                        ui.label("Hello World!");
                    });
```
creates a window with the resize-handle, but I can't resize it - nor by the handle nor by dragging the window edge. If I add `scroll(true)` then I can resize the window horizontally. But not vertically. Similar behavior can be seen in the web demo `Window Options` - despite the fact the checkbox `resizable` is selected the actual resize happens only horizontally. In the demo `Manual Layout Test` the window is not resizable at all though the property is not set to false explicitly. I'm looking at the source code for this example but can't figure out why it behaves so.

**Comment by emilk** [comment]
Windows in egui shrink/expand to fit the contents. If there is nothing in the window, there will be nothing to show.

See the `Window Resize` demo at https://emilk.github.io/egui/

If you want a blank area, you can allocate some dead space with `ui.allocate_space(ui.available_size());`
